packages/transaction-log/src/services/api.service.ts

```
@@ -80,3 +81,47 @@ export async function getServerHeartBeat(): Promise<{
+  try {
+    await addToTaskQueue(params)
+  } catch (error) {
+    throw error
+  }
+}
+
+export async function getBulkRunFilterCount(params: {
+  filterLimit: number
+  filterPage: number
+  filterRepresentation: string
+  filters: {}
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)